src/xdocs/writingchecks.xml

```diff
@@ -64,17 +64,17 @@
 
       <p>
         Suppose you have a convention that the number of methods in a
-        class should not exceed a certain limit, say 30. This rule makes
-        sense, a class should only do one thing and do it well. With a
+        class should not exceed 30. This rule makes
+        sense: a class should only do one thing and do it well. With a
```

Comment:
Again, I am not really liking the `:`s. Either an exclamation or a period.

Reply:
Fixed